tools/maxquant/maxquant.xml

```diff
@@ -89,7 +89,16 @@
             lfqStabilizeLargeRatios: ${protein_quant.lfq_opts.lfqStabilizeLargeRatios}
             lfqRequireMsms: ${protein_quant.lfq_opts.lfqRequireMsms}
             advancedSiteIntensities: ${protein_quant.lfq_opts.advancedSiteIntensities}
-            matchBetweenRuns: ${search_opts.match_between_runs}
+            #if $search_opts.mbr.match_between_runs == 'True':
+            matchBetweenRuns: ${search_opts.mbr.match_between_runs}
+            matchingTimeWindow: ${search_opts.mbr.matching_time_window}
+            matchingIonMobilityWindow: ${search_opts.mbr.matching_ion_mobility_window}
+            alignmentTimeWindow: ${search_opts.mbr.alignment_time_window}
+            alignmentIonMobilityWindow: ${search_opts.mbr.alignment_ion_mobility_window}
+            matchUnidentifiedFeatures: ${search_opts.mbr.match_unidentified_features}
+            #else:
+            matchBetweenRuns: ${search_opts.mbr.match_between_runs}
```

Comment:
why not taking it out of the if, as before?

Reply:
`matchBetweenRuns: False` is missing in the config.yml file where it was before the update and therefore Planemo raises a lines_diff error.

Reply:
What I mean is that this line is both in the if **and** else.

Reply:
True! Sorry, it's way too late.... ;)